### PR TITLE
fix handleSubmitPreCommitBatch() preCommitParams error

### PR DESCRIPTION
**Pull request by johnli-helloworld**
preCommitParams() will send event ， and two events will be sent.

### CODE REVIEW

File: extern/storage-sealing/states_sealing.go [hunk 1]
```diff
@@ -360,7 +360,7 @@ func (m *Sealing) handleSubmitPreCommitBatch(ctx statemachine.Context, sector Se
 
 	params, deposit, _, err := m.preCommitParams(ctx, sector)
 	if params == nil || err != nil {
-		return ctx.Send(SectorChainPreCommitFailed{xerrors.Errorf("preCommitParams: %w", err)})
+		return err
```
Comment (magik6k) [inline extern/storage-sealing/states_sealing.go:363]: I think it's more correct to still send the event if we see the error
```suggestion
		if err != nil {
			return ctx.Send(SectorChainPreCommitFailed{xerrors.Errorf("preCommitParams: %w", err)})
		}
		return nil // event was sent in preCommitParams
```

**Review by magik6k — COMMENTED** [review]
Thanks for the PR, just 1 comment.

There is also the same `if` in `handlePreCommitting`, should fix that one too.

**Comment by jennijuju** [comment]
@johnli-helloworld  - Could you please see Magik's comment and address it accordingly?

**Comment by magik6k** [comment]
Merged in https://github.com/filecoin-project/lotus/pull/6763, thanks!

**Pull request closed without merging**